app/services/curation_concerns/workflow/confirm_submission.rb

```diff
@@ -0,0 +1,30 @@
+module CurationConcerns
+  module Workflow
+    class ConfirmSubmission
+      # params consists of
```

Comment:
Please use the YARD doc style for this. See http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option